.travis.yml

```diff
@@ -28,6 +26,8 @@ addons:
       - libglew-dev
       - libopenni-dev
       - python3-pip
+      - python3-sphinxcontrib.doxylink
+      - libproj-dev #missing dependency from vtk?
```

Comment:
Probably #1594, I would vote for trying bionic ;).

Reply:
I know that xenial is in a kind of "beta" but I haven't seen anything with bionic. Doesn't hurt to try and see what happens.

Reply:
```yaml
dist: bionic
```
is not supported and defaults to precise.  We'll only be able to try it out this once we start fiddling with the docker approach.

Reply:
Ah, yeah was talking about docker...